### PR TITLE
Remove organic from products

### PR DESCRIPTION
**Pull request by ErikOstien**
This addresses the chore to move organic into certifications, and includes a new migration to remove organic, removes organic from view, controller, and factory.

### CODE REVIEW

**Comment by dwang733** [comment]
Looks good, but I think you should also remove the organic field from the product factory.

File: db/schema.rb [hunk 1]
```diff
@@ -78,17 +78,10 @@
     t.datetime "updated_at", null: false
     t.text "origin"
     t.text "cultural_history"
-    t.boolean "organic"
     t.string "picture"
     t.index ["vendor_id"], name: "index_products_on_vendor_id"
   end
 
-  create_table "tags", force: :cascade do |t|
```
Comment (jlabowitz) [inline db/schema.rb:86]: Why is this table being deleted?

File: db/schema.rb [hunk 2]
```diff
@@ -98,15 +91,6 @@
     t.index ["vendor_id"], name: "index_vendor_ownerships_on_vendor_id"
   end
 
-  create_table "vendor_tags", force: :cascade do |t|
```
Comment (jlabowitz) [inline db/schema.rb:101]: Same with this table, why is this being deleted?

Comment (ErikOstien) [reply db/schema.rb:101]: It seems these were removed in the new migration. David says these tables should have been removed a while ago, as they are not in use anymore. So this should be fine.

**Pull request merged**